Packs/Cisco-umbrella-cloud-security/Integrations/Cisco-umbrella-cloud-security/Cisco-umbrella-cloud-security.py

```diff
@@ -1,12 +1,15 @@
+import base64
+import json
+import traceback
+
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 
-''' IMPORTS '''
+register_module_line('Cisco Umbrella Cloud Security', 'start', __line__())
 
 
-import base64
-import json
-import traceback
+''' IMPORTS '''
+
```

Comment:
Please remove

Reply:
Removed. 

Do you know why this "register_module_line" can happen? It has already happened to me with another integration when I contribute through the marketplace.

Thanks!

Reply:
For some reason contributing throw the marketplace adds this line.